app/src/main/java/com/shub39/rush/domain/dataclasses/TTMLParser.kt

```
@@ -319,26 +324,18 @@ object TTMLParser {
-                val agentPrefix = if (!line.agent.isNullOrEmpty()) "{agent:${line.agent}}" else ""
-
                 appendLine(
-                    String.format(
-                        "[%02d:%02d.%02d]%s%s",
-                        minutes,
-                        seconds,
-                        centiseconds,
-                        agentPrefix,
-                        line.text,
-                    )
+                    String.format("[%02d:%02d.%02d]%s", minutes, seconds, centiseconds, line.text)
```

detekt (1.23.8)

[warning] String.format("[%02d:%02d.%02d]%s", minutes, seconds, centiseconds, line.text) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)